mkdocs.yml

```diff
@@ -50,8 +51,8 @@ nav:
           - Notable Contributors: notable-contributors.md
     - Guides:
           - Pre-Install: guides/preinstall.md
+          - Basic setup: guides/setting-up.md
```

Comment:
possibly consider "postinstall" as the name, but "setting up" is fine too

Reply:
Done